.drone.yml

```diff
@@ -699,6 +699,8 @@ steps:
   - cd $TeleportSrc
   - git submodule update --init e
   - git submodule update --init --recursive webassets
+  - cd $WebappsSrc
+  - git submodule update --init packages/webapps.e
```

Comment:
I would like to see the drone changes get their own PR so that any potential issue can more easily be pinpointed or rolled back, and the backport decision can be independent